react-accordion-component/my-app/src/Accordion.js

```diff
@@ -0,0 +1,36 @@
+import './Accordion.css';
+
+/**
+ *
+ * @param {topics} An array of objects. Each object will contain a header and corresponding content for the Accordion panel.
+ */
+
+export default function Accordion ({topics, view, onSwapView}) {
+  return (
+    <div className="all-panels">
+      {topics.map((topic)=> (
+        <Topic className="panel"
+          view={view}
+          key={topic.header}
+          header={topic.header}
+          content={topic.content}
+          onSwapView={onSwapView} />))}
+    </div>
+  )
+}
+
+function Topic({view, header, content, onSwapView}){
```

Comment:
I would prefer `isOpen` or `isExpanded` to `view`, which doesn't have a lot of meaning.

You also are requiring the 'view' to have the exact same value as 'header' when it's being displayed. That is also a bigger burden than is necessary.